ui/Screen/Profile/Following.svelte

```diff
@@ -55,6 +55,7 @@
     display: flex;
     align-items: center;
     justify-content: space-between;
+    border-radius: 0.5rem;
```

Comment:
Shouldn't we do this by doing `overflow: hidden;` on the container?

Reply:
🙇

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/1917/commits/604f8c9d0852cd2c292f4dba04b7498fc2b6ad67